src/rules/selector-max-specificity/README.md

```diff
@@ -12,7 +12,7 @@ Visit the [Specificity Calculator](https://specificity.keegan.st) for visual rep
 
 This rule ignores selectors with variable interpolation (`#{$var}`, `@{var}`, `$(var)`).
 
-This rule ignores selectors containing the `:not()` pseudo-class.
+This rule ignores selectors containing the `:not()` or `:matches()` pseudo-class.
```

Comment:
What about `:unknown()`, above? Is there a more general statement we should make, like it ignores all pseudo-classes with functional notation, *such as* `:not()` and `:matches()`?